packages/safe-core-sdk-types/src/contracts/MultiSendCallOnlyContract.ts

```diff
@@ -0,0 +1,4 @@
+export interface MultiSendCallOnlyContract {
+  getAddress(): string
+  encode(methodName: any, params: any): string
```

Comment:
Is using `any` necessary?

Reply:
Otherwise it gets too complex. It is one of those cases where the type of one parameter depends on the value of another parameter. There are a few similar cases we discussed in the past